scripts/services/emoncms_mqtt/emoncms_mqtt.php

```diff
@@ -279,7 +279,7 @@ function message($message)
                             $time = $timestamp;
                         }
                     } else {
-                        $log->warn("Time value not valid ".$inputtime);
+                        $log->warn("Time value not valid ".json_encode($inputtime));
```

Comment:
What difference does the `.json_encode` make?

Reply:
Because $inputtime is derived from json_decoded values it can be a non-scalar type, an array or whatever, and attempting to print it was killing the whole script.